src/ert/cli/workflow.py

```diff
@@ -10,7 +10,7 @@ def execute_workflow(ert, workflow_name):
         msg = "Workflow {} is not in the list of available workflows"
         logger.error(msg.format(workflow_name))
         return
-    context = workflow_list.getContext()
+    context = ert.getDataKW()
```

Comment:
is `getDataKW()` equivalent to `getContext()` ?

Reply:
yea, it seems like that when i'm poking around. Would it be a disaster actually renaming this to something sane? Or do we end up breaking some api? Mixing context, getDataKW and SubstitutionList here - not making any sense for me the names....  Might just be only me confused though, if so - nevermind - i can live with it :sweat_smile:

Reply:
Seems that renaming `getDataKW` to `get_context` does not break any API that I could find and will clear some of the name overloading for the SubstitutionList